recipes/tldr/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "tldr" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 731b10bf38bb84a73560408a783b26d08ac52eab2e9a55f89dba9a3c23a72892
+
+build:
+  number: 0
```

This looks to be pure Python and thus able to be noarch Python?
```suggestion
  number: 0
  noarch: python
```